src/kvstore.h

```diff
@@ -1,85 +1,90 @@
-#ifndef DICTARRAY_H_
-#define DICTARRAY_H_
+#ifndef KVSTORE_H
+#define KVSTORE_H
 
-#include "dict.h"
+#include "hashset.h"
```

Comment:
I think you forgot to include this file

Reply:
This PR currently targets the `hashset` feature branch, where this file exists.

Do you think I should target the unstable branch instead? I'm trying to find a way to split it into smaller PRs...

Reply:
How about we first create a PR to unstable with just the hashset implementation and replace dict with hashet in the command lookup table?

Reply:
Oh, I missed this, I thought this was the PR for the hashset and I was interested. You can ignore me :D

Reply:
No, seriously, I think it can be very hard to get a huge feature branch merged. We should split it up in some way.

The implementation of the hashset is in the hashset branch in this repo. PTAL!